pkg/gateway/api/api_test.go

```diff
@@ -1983,7 +1983,7 @@ func TestRegisterEventsContextCanceled(t *testing.T) {
 }
 
 func TestRegisterEvents(t *testing.T) {
-	t.Parallel()
+	// 	t.Parallel()
```

Comment:
Please let me know if there is a reason not to use Parallel on this test.

Reply:
I forgot to remove it. Thx.